docs/reference/commandline/create.md

```diff
@@ -69,6 +69,7 @@ Creates a new container.
       -P, --publish-all=false       Publish all exposed ports to random ports
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
+      --pids-limit=0                Tune container pids limit (set -1 for unlimited)
```

Comment:
@jfrazelle why 512? When I ran tests I noticed that even a small number was enough to significantly slow down (and in some cases completely crash) the host machine. Just wondering if we should try to empirically test what a reasonable host is able to withstand without bringing down the host.